### PR TITLE
Add Go version to upstream projects tracker

### PR DESCRIPTION
**Pull request by abhay-krishna**
This change moves the Go version definition from the project Makefile to a separate tracking file. The Go version for a project has also been added to the upstream projects tracker file.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/apache/cloudstack-cloudmonkey/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
 GIT_TAG:=$(shell cat GIT_TAG)
-GOLANG_VERSION?="1.20"
+GOLANG_VERSION?=$(shell cat GOLANG_VERSION)
```
Comment (jaxesn) [inline projects/apache/cloudstack-cloudmonkey/Makefile:3]: can you replace all of these with `:=` instead of `?=`

**Review by jaxesn — CHANGES_REQUESTED** [review]
I like this!

We didnt have to split the versions into a separate file, we could have called make to get the golang version like we do in our buildspec generation scripts.  This is a pattern we have followed tho for release branch'd builds so im fine with the change.

**Comment by abhay-krishna** [comment]
> I like this!
> 
> We didnt have to split the versions into a separate file, we could have called make to get the golang version like we do in our buildspec generation scripts.  This is a pattern we have followed tho for release branch'd builds so im fine with the change.

True, the reason I split them even for non release branched projects is to make the automation for Git tags/go version updates cleaner. Felt it would be better to update the single tracking file instead of invoking sed on the Makefile.

**Review by jaxesn — APPROVED** [review]

**Pull request merged**